packages/typescript-estree/tests/lib/parse.test.ts

```diff
@@ -40,11 +40,11 @@ jest.mock('typescript', () => {
   };
 });
 
-jest.mock('fast-glob', () => {
-  const fastGlob = jest.requireActual<typeof fastGlobModule>('fast-glob');
+jest.mock('tinyglobby', () => {
+  const tg = jest.requireActual<typeof tinyglobby>('tinyglobby');
   return {
-    ...fastGlob,
-    sync: jest.fn(fastGlob.sync),
+    ...tg,
+    globSync: jest.fn(tinyglobby.globSync),
```

Comment:
Previously the thing being passed to `jest.fn` was from the `jest.requireActual` object. So to keep to that:

```suggestion
    globSync: jest.fn(tg.globSync),
```

Reply:
Thanks, I totally missed that!